src/parser.js

```diff
@@ -48,8 +48,8 @@ function getLastExpression(code) {
   const minified = (code || '')
     // remove comments
     .replace(/\/\*[\s\S]*?\*\/|\/\/.*/g, '')
-    // remove all white space
-    .replace(/\s/g, '');
+    // remove all white space outside quotes
+    .replace(/[ ]+(?=[^"\'`]*(?:["\'`][^"\'`]*["\'`][^"\'`]*)*$)/g, '');
```

Comment:
Nice try, but this will fail on nested quotes:

```
'container.querySelector("I\'ll `be back`")'
  .replace(/[ ]+(?=[^"\'`]*(?:["\'`][^"\'`]*["\'`][^"\'`]*)*$)/g, '');

> "container.querySelector('"I'llbeback"')"
```

Reply:
I think that for this case we can't use the regex. I think that we can create a function to do this

Reply:
Or we need to rethink on how to extract the last expression, so we don't need this white-space trimming.

Let's merge this first. It's definitely better than what we currently have.

Agreed?